package.json

```diff
@@ -18,15 +18,17 @@
     "test": "npm run test:e2e:ci",
     "test:unit": "vue-cli-service test:unit",
     "commit": "git-cz",
-    "lint": "vue-cli-service lint",
+    "lint": "eslint src tests build --ext .ts,.tsx,.vue,.js,.jsx",
```

Comment:
So for what I saw previously there were some files that were not lined right? Good catch

Reply:
yes, I copied the one from x-components because here we were not checking all of them.